packages/commonwealth/client/scripts/views/components/sidebar/index.tsx

```diff
@@ -23,9 +30,18 @@ type SidebarAttrs = {
 export class Sidebar implements m.ClassComponent<SidebarAttrs> {
   view(vnode: m.VnodeDOM<SidebarAttrs, this>) {
     const { isSidebarToggleable, isSidebarToggled } = vnode.attrs;
-
-    const hideChat = !app.chain?.meta?.chatEnabled;
-
+    const activeAddressRoles = app.roles.getAllRolesInCommunity({
+      chain: app.activeChainId(),
+    });
+    const currentChainInfo = app.chain?.meta;
+    const hideChat =
```

Comment:
`hideChat` should be biased toward hiding rather than showing, e.g. how the older version returned true if chain/meta were undefined, as well as based on the variable.